### PR TITLE
feat(mixins): export mixins

### PR DESCRIPTION
**Pull request by talkor**
https://monday.monday.com/boards/3532714909/pulses/4982541657

I haven't changed the mixins themselves but probably a small refactor next might be beneficial

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,14 +84,31 @@ To use the supplied config, add `monday-ui-style/stylelint-config` as a [Styleli
...
+- `map-collect`: merges maps from different scopes into one
+- `extract-rgb`: returns a comma separated list of rgb values from a color
 
 ## Mixins
-We exporting the following scss mixins:
-- vibe-heading(type, weight)
-- vibe-text(type, weight)
+
+The following SCSS mixins are exported:
+
```
Comment (SergeyRoyt) [inline README.md:99]: Typography:

File: README.md [hunk 2]
```diff
@@ -84,14 +84,31 @@ To use the supplied config, add `monday-ui-style/stylelint-config` as a [Styleli
...
+- `vibe-heading($type, $weight)`
+- `vibe-text($type, $weight)`
+- `line-clamp($lines)`
+- `ellipsis`
+- `disabled`
+- `focus-style($focus-radius)`
+- `scroller($width, $height, $color)`
+- `grid-auto-fit($grid-gap, $grid-column-gap, $grid-row-gap. $grid-cell-min-width, $grid-cell-array-calc, $important)`
+- `reset-button`
+- `hidden-element`
```
Comment (SergeyRoyt) [inline README.md:109]: Let's split them into common and states according to files

File: src/mixins/_common.scss [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  box-shadow: var(--box-shadow-small);
+}
+
+@mixin box-shadow-medium() {
+  box-shadow: var(--box-shadow-medium);
+}
+
+@mixin box-shadow-large() {
+  box-shadow: var(--box-shadow-large);
+}
```
Comment (SergeyRoyt) [inline src/mixins/_common.scss:23]: Looking at them makes me question if we need them at all... Maybe let's remove them, wdyt?

Comment (talkor) [reply src/mixins/_common.scss:23]: Exactly what I thought, I agree

**Review by SergeyRoyt — APPROVED** [review]
Looks good!
Let's also create a PR in vibe which replaces these mixins with imports from monday-ui-style

**Pull request merged**